Zend/zend_strtod.c

```diff
@@ -159,7 +159,6 @@
  *	floating-point numbers and flushes underflows to zero rather
  *	than implementing gradual underflow, then you must also #define
  *	Sudden_Underflow.
- * #define USE_LOCALE to use the current locale's decimal_point value.
```

Comment:
I accidentally committed this change... But if I'm right then `USE_LOCALE ` is always undefined (see Zend/zend_strtod_int.h) so there's no need for supporting locale-sensitivity.

Reply:
Is it necessary to touch the zend_strtod code at all? This is a bundled library.

Reply:
No, it's absolutely not necessary. I only touched it at the very beginning when I wasn't sure what to do :) That said, I'll get rid of this change.